config/volume.go

```diff
@@ -343,35 +347,34 @@ func (c *Client) WatchSnapshotSignal(activity chan *watch.Watch) {
 	watch.Create(w)
 }
 
-// Validate options for a volume. Should be called anytime options are
-// considered.
-func (ro *RuntimeOptions) Validate() error {
-	if ro.UseSnapshots && (ro.Snapshot.Frequency == "" || ro.Snapshot.Keep == 0) {
-		return errored.Errorf("Snapshots are configured but cannot be used due to blank settings")
-	}
-
-	return nil
-}
-
-// Validate validates a volume configuration, returning error on any issue.
+// Validate validates a volume configuration, returning error on any
```

Comment:
I was referring to the deletion of " issue."  The sentence is incomplete in the latest commit.  It looks like the last word was accidentally deleted.